tests/indices/embedding/test.py

```diff
@@ -0,0 +1,129 @@
+"""Test embedding functionalities."""
```

Comment:
as discussed, let's create simpler unit tests, but we should def move the text files as part of an example notebook